language_formatters_pre_commit_hooks/ktlint.version

```diff
@@ -1 +1 @@
-0.41.0
+0.40.0
```

Comment:
Should you amend this?

Reply:
check the commit message
Essentially with the latest ktlint tests are failing so I'm downgrading it again.
in the meantime the changes in this pr should make sure that next time auto update happens tests will be run so we won't end up merging them if tests are red

Reply:
Oh my bad haven't seen that. Then LGTM 👍